site2/docs/concepts-messaging.md

```diff
@@ -149,6 +149,12 @@ Messages can be acknowledged in the following two ways:
 - Messages are acknowledged individually. With individual acknowledgement, the consumer needs to acknowledge each message and sends an acknowledgement request to the broker.
 - Messages are acknowledged cumulatively. With cumulative acknowledgement, the consumer only needs to acknowledge the last message it received. All messages in the stream up to (and including) the provided message are not re-delivered to that consumer.
 
+The following example shows how to call the API :
+
+```
+consumer.acknowledge(msg);
+```
```

Comment:
Should we also add the example for cumulative acknowledgement here?

Reply:
My original idea is to let the beginners know that they are going in different ways, because I was confused about this before ....

Reply:
No problem. So why not add the example for cumulative acknowledgement here? It could be weird that the description contains two ways of acknowledgement but the code only shows the first way.

Reply:
Agree with @BewareMyPower, can you add an example for cumulative acknowledgement? @yonyong

Reply:
I think you have a point，I have pushed the code @Anonymitaet @BewareMyPower